runbotics/runbotics-orchestrator-ui/src/src-app/views/process/EditProcessDialog/AccessOptions/AccessOptions.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { FC } from 'react';
+
+import { Box, Switch, FormControlLabel } from '@mui/material';
+
+import { CollectionId } from 'runbotics-common';
+
+import Accordion from '#src-app/components/Accordion';
+import useTranslations from '#src-app/hooks/useTranslations';
+
+import InfoButtonTooltip from '#src-app/views/process/ProcessBuildView/Modeler/ActionFormPanel/widgets/InfoTooltip/InfoButtonTooltip';
+
+import { AccessOptionsProps } from './AccessOptions.types';
+import LocationOptions from './LocationOptions';
+
+
+const AccessOptions: FC<AccessOptionsProps> = ({ processData, setProcessData, isOwner, isEditDialogOpen }) => {
+    const { translate } = useTranslations();
+    const isPublicTranslationKey = processData.isPublic ? 'True' : 'False';
+
+    const changeCollectionId = (newId: CollectionId) => {
+        setProcessData((prevState) => ({
+            ...prevState,
+            processCollection: {
+                // ...prevState.processCollection, // todo
```

Comment:
Is it ommited to implement in future, or forgotten? :) Or in this version change of process collection id is enough?

Reply:
forgotten. Already implemented the feature, thanks